packages/gatsby-plugin-offline/src/gatsby-browser.js

```diff
@@ -28,8 +28,12 @@ exports.onServiceWorkerInstalled = ({ getResourceURLsForPathname }) => {
     .call(nodes)
     .map(node => node.src || node.href || node.getAttribute(`data-href`))
 
+  const a = document.createElement(`a`)
+
   for (const resource of resources) {
-    fetch(resource)
+    a.href = resource
```

Comment:
It would be better to use `const a = new URL(resource)` here - same effect but we don't need to create a hidden element